src/test/librgw_file.cc

```diff
@@ -61,8 +61,6 @@ TEST(LibRGW, MOUNT) {
 
 TEST(LibRGW, GETATTR_ROOT) {
   if (do_getattr) {
```

Comment:
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes. in this case, it'd be `librgw_file: `.